recipes/ipopt/meta.yaml

```diff
@@ -0,0 +1,46 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "Ipopt" %}
+{% set version = "3.12.5" %}
```

Comment:
No, I think this was just the version pulled by a recipe I was basing this off. I can definitely update it, I'll need to re-pull from http://www.coin-or.org/download/pkgsource/ instead of /source/ so we don't build third-party libs. Still waiting for #1778 to go through though.